suite-common/message-system/config/config.v1.json

```diff
@@ -90,6 +90,63 @@
                 "context": { "domain": "accounts.eth.staking" }
             }
         },
+        {
+            "conditions": [
+                {
+                    "settings": [
+                        {
+                            "matic": true
+                        }
+                    ],
+                    "environment": {
+                        "desktop": "*",
+                        "mobile": "!",
+                        "web": "*"
+                    }
+                }
+            ],
+            "message": {
+                "id": "c8ca8958-7868-467f-9278-f072edc234ac",
+                "priority": 93,
+                "dismissible": true,
+                "variant": "info",
+                "category": "banner",
+                "content": {
+                    "en-GB": "Polygon PoS will migrate from MATIC to POL on September 4. Your balance will appear as MATIC until an update is available.",
+                    "en": "Polygon PoS will migrate from MATIC to POL on September 4. Your balance will appear as MATIC until an update is available.",
```

Comment:
```suggestion
                    "en": "Polygon PoS will migrate from MATIC to POL on September 4, no action is required. Your balance will appear as MATIC until an update is available.",
```

Reply:
Solved in 1da9e69075e68bfbe3d3bc796a6adfb478c07969